### DISCUSSION TITLE
webp in PHP 5.6

### DISCUSSION THREAD

**Discussion by jmpw** [shivammathur/homebrew-php#716] [Q&A / Help] [answered]
May I ask you how to add webp image support to 5.6 version (got `Call to undefined function imagewebp()` error)?

**Comment by shivammathur** [comment]
PHP 5.6 is not built with webp support. I would suggest using PHP 7.0 or newer for it.

**Comment by jmpw** [comment, marked as answer]
Thank you for your answer. Unfortunatelly, in this project we have to use version 5.6. However, I manually updated `php@5.6.rb`, added webp and libvpx dependencies and build PHP from source. Luckily, it works.

**Reply by shivammathur** [reply]
Yes, building with `libvpx` will add support for `webp`, but I'm wary of adding it as brew provides the latest version of `libvpx`, and webp support using it in PHP 5.6 is 11 years old. Hopefully, it works for you.
